gallery/lib/l10n/README.md

```diff
@@ -45,7 +45,7 @@ dart ${YOUR_FLUTTER_PATH}/dev/tools/localization/bin/gen_l10n.dart \
     --output-class=GalleryLocalizations
 ```
 
-From `samples/gallery/gallery/`, run `dart ../l10n_cli/bin/main.dart`, which
+From `samples/gallery/`, run `dart ../l10n_cli/bin/main.dart`, which
```

Comment:
I think `dart ../l10n_cli/bin/main.dart` should become `dart l10n_cli/bin/main.dart`

Reply:
Good catch